contracts/tests/strategies/DnsVectorStrategyV2.sol

```diff
@@ -0,0 +1,354 @@
+// SPDX-License-Identifier: BUSL-1.1
```

Comment:
Thanks for adding this, but I think we might not want to do tests like this in the future. 
I feel the only thing here we are testing is if we inherited from an 'upgareable' base class. We might just change our simple strategies like Cash to non upgradeable in the future. I feel the usefulness of this test is not justifying the effort to test 'upgadeablility'.  

Thanks to you, we already have a great way to test upgradebility for already deployed live contracts. Those tests make sure that the potential new data layout of the contract is not going to cause us problems.

Reply:
Upgrading, let's say, Cash to OwnableStrategyV2 cause data layout problem. That's why when we test upgradeability, we need to upgrade to corresponding V2 contracts.

Reply:
but in the tests we just:
1. copy let's say DnsVectorStrategy and name it DnsVectorStrategyV2
2. test if DnsVectorStrategy  can be upgraded to DnsVectorStrategyV2 which is always the case as long as DnsVectorStrategy is upgradeable (the data layout is the same)

Is there any scenario where this test can be useful?

Reply:
These tests' original purpose is to check if data layout AND AUM are intact. But since I already added upgrade after deploy tests so testing if it's upgradeable is sufficient. I'll remove unnecessary parts.

Reply:
Great!